src/pages/nominate.js

```diff
@@ -52,16 +52,13 @@ class App extends React.Component {
 
   handleSubmit = e => {
     const validation = this.validateName();
+    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to send your nomination')) {
+      const urlEncodedTwitterHandle = encode(this.state.name);
+      const urlEncodedReason = encode(this.state.reason);
+      const directMessageLink =
+        `https://twitter.com/messages/compose?text=Please+consider+following+${this.state.name}+because+${this.state.reason}&recipient_id=1129894370339983360`
```

Comment:
I'm not sure if it's possible, but can the `recipient_id` be dynamic?  It's loaded in gatsby-config.js from `process.env.TWITTER_ID_FOR_FOLLOWING_LIST` into `twitterIdForFollowingList`, so maybe it's available in that variable here as well?

I don't think it's a blocker since it's less about data hiding and more about making it easier for others to use this feature, so if it doesn't seem possible, no problem

Reply:
Hmm it looks like process.env is always empty. I think that's because this page is only using the browser javascript environment and not the server-side one. I could change it to a variable declared at the top of the scope like:

```
const TWITTER_ID_FOR_FOLLOWING_LIST = '1129894370339983360'
const directMessageLink = `https://twitter.com/messages/compose?text=Please+consider+following+${this.state.name}+because+${this.state.reason}&recipient_id=${TWITTER_ID_FOR_FOLLOWING_LIST}`
```

I think that could help make it more explicit that the id needs to be changed if anyone else forks the projects.